core/src/main/scala/design/hamu/core/command/DecimalToBinaryCommand.scala

```diff
@@ -0,0 +1,28 @@
+package design.hamu.core.command
+
+object DecimalToBinaryCommand extends Command {
+
+  def run(ls: List[String]): String = ls match {
+    case decimal :: tail =>
+      var isNegative = false
+      val abs = (decimal: Int) => {
+        if (decimal >= 0) {
+          decimal
+        } else {
+          isNegative = true
```

Comment:
Another example of bad mutability/side effects :)
Also you can just use `Math.abs` here since I think `abs` is pretty trivial implementation.